frontend/src/components/notImplemented/NotImplemented.js

```diff
@@ -0,0 +1,24 @@
+import React,{ useState } from 'react';
+import {Redirect} from 'react-router-dom';
+import './notImplemented.css';
+
+const NotImplemented = () => {
+
+    const [authenticated, setAuthenticated] = useState(true);
+
+    const redirect = () => {
+        localStorage.clear();
+        setAuthenticated(false);
+    };
+
+    if (authenticated) return (
+        <div className='wrapper'>
+            <p>Not Implemented yet.</p>
+            <button onClick={ redirect }>Go Home</button>
```

Comment:
@gulyaasjaanos this is not a Go Home, but a Logout button :)
either the button should say `Logout and go to Login` or it should just redirect to the kingdom page if the user is logged in